lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -124,6 +124,28 @@ public function get_items( $request ) {
 			unset( $args['filter'] );
 		}
 
+		if ( isset( $request['sticky'] ) ) {
```

Comment:
As noted in slack, `sticky=false` does not work (it's interpreted as `true` due to string coercion -- suggest using `rest_is_boolean` introduced in #2630